src/NavigationState.ts

```diff
@@ -7,16 +7,22 @@ interface State {
   stackCounter: number;
   activeScreenId?: string;
   activeComponentName?: string;
+  activeTabIndex: number;
   isActiveScreenId: (screenId: string) => boolean;
 }
 
 const state: State = {
   stackCounter: 0,
   activeScreenId: undefined,
+  activeTabIndex: 0,
```

Comment:
Is it right to say the initial index is 0? 
Is it possible the initial tab will be something else?

Reply:
i think so, it's the default tab.
i don't know about a way to set a different tab as default 🤔

Reply:
Ok, let's keep it like that. I don't expect it to be a problem any time soon..